.github/workflows/preview-documentation.yml

```diff
@@ -47,6 +46,12 @@ jobs:
           yarn install --immutable
           yarn build:fast
 
+  deploy:
```

Comment:
How does this solve the issue? Is this somehow guaranteed to be executed serially among all PRs?

Reply:
Notice the `concurrency`-argument below. This ensure there only will be one execution of the step across all ongoing Github Actions.

Reply:
Alright, seems good :+1: